client/components/product-icon/test/test.jsx

```diff
@@ -0,0 +1,104 @@
+jest.mock( 'lib/abtest', () => ( {
+	abtest: () => '',
+} ) );
+
+jest.mock( 'i18n-calypso', () => ( {
+	localize: Comp => props => (
+		<Comp
+			{ ...props }
+			translate={ function( x ) {
+				return x;
+			} }
+		/>
+	),
+	numberFormat: x => x,
+	translate: x => x,
+} ) );
+
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
```

Comment:
Any good reason to not use `jest`'s built-in assertions (`expect`) and use `chai` instead?

Reply:
I copied the whole test file from `PlanIcon` and haven't changed it at all. It's probably a legacy approach. I'm going to change this.

Reply:
Yeah, some of those tests were never migrated to Jest. But it's recommended that we use it for any new tests.